tests/system/deadServiceRecreated_test.go

```diff
@@ -0,0 +1,29 @@
+package system
+
+import (
+	"testing"
+	"time"
+)
+
+// Test Resources:
+// This test creates an environment named 'dsr' that has a
```

Comment:
yeah, it's just an acronym of the test case name. This was added into the readme